### DISCUSSION TITLE
Parse multi-line text to create sub-stasks

### DISCUSSION THREAD

**Discussion by Molin-L** [mgmeyers/obsidian-kanban#168] [Ideas & Feature Requests]
In many cases, we split one task into several sub tasks. So that we can have good vision on what these task for. Github project is doing great. I think there are many funcions we can refer from. BTW, you're doing a good job. It really helps me to organize my work usinfg Obsidian. 🥰

**Comment by samuelskeirik** [comment]
Love this plugin. I agree, subtasks by multiline would be fantastic.

**Reply by bodiequirk** [reply]
Agreed! I would love the ability even to use SHIFT + RETURN to do a soft return and make small list on separate lines.
